checkpoint/cp-resource-management/templates/rolebinding.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.pspEnable }}
```

Comment:
I think you need to add {{ if .Values.rbac.create }} and then you don't need it in the subject

Reply:
On review, I don't agree with this comment